clients/go/internal/containersuite/containerSuite.go

```diff
@@ -90,6 +93,24 @@ func (s zeebeWaitStrategy) WaitUntilReady(ctx context.Context, target wait.Strat
 	}
 }
 
+// remove the message header (8 bytes) from the docker logs
+// https://docs.docker.com/engine/api/v1.26/#operation/ContainerAttach
+func sanitizeDockerLogs(log string) string {
+	lines := strings.Split(log, "\n")
```

Comment:
we split it then rejoin it? can't we just to `log[8:]`? we just want to remove the first 8 bytes right?

Reply:
That would remove the first 8 bytes in the entire log, leaving the headers in the other lines no?

Reply:
Ah, right, my bad, :+1: